base/build/pysgpp/base.i

```diff
@@ -28,6 +28,7 @@
 %shared_ptr(sgpp::parallel::OperationParabolicPDESolverSystemFreeBoundaries)
 
 %include "base/src/sgpp/globaldef.hpp"
+%include "std_set.i"
```

Comment:
```suggestion

```

Reply:
Fine by me.
@WeberSamuel, can you commit this? Then, jenkins will hopefully run through all the way.